pkg/machine/ocipull/ociartifact.go

```diff
@@ -165,6 +165,7 @@ func (o *OCIArtifactDisk) get() error {
 }
 
 func (o *OCIArtifactDisk) getDestArtifact() (types.ImageReference, digest.Digest, error) {
+	fmt.Println("Looking up Podman Machine image to create VM")
```

Comment:
I think this should include the actual image name in the message.

Reply:
Can you propose a format?  Should the name be in brackets, quoted, etc?

Reply:
`"Looking up %q image to create VM", o.imageEndpoint` maybe?

Reply:
I played a bit and liked the following the most:
```diff
diff --git a/pkg/machine/ocipull/ociartifact.go b/pkg/machine/ocipull/ociartifact.go
index 5824944a8cfc..edb6f4e6eaf9 100644
--- a/pkg/machine/ocipull/ociartifact.go
+++ b/pkg/machine/ocipull/ociartifact.go
@@ -165,11 +165,11 @@ func (o *OCIArtifactDisk) get() error {
 }
 
 func (o *OCIArtifactDisk) getDestArtifact() (types.ImageReference, digest.Digest, error) {
-       fmt.Println("Looking up Podman Machine image to create VM")
        imgRef, err := alltransports.ParseImageName(o.imageEndpoint)
        if err != nil {
                return nil, "", err
        }
+       fmt.Printf("Looking up Podman Machine image at %s to create VM\n", imgRef.DockerReference())
        sysCtx := &types.SystemContext{
                DockerInsecureSkipTLSVerify: types.NewOptionalBool(!o.pullOptions.TLSVerify),
        }
```